### ISSUE TITLE
False positive in Lint/Debugger 

### ISSUE THREAD

**Issue by jcoyne** [rubocop/rubocop#11556]
There is a regression caused by #11555, where it thinks that any use of the local variable `p` is a method call to the debugger.

Example:
```
spec/views/spotlight/sir_trevor/blocks/_solr_documents_features_block.html.erb_spec.rb:49:23: W: Lint/Debugger: Remove debugger entry point p.
      render partial: p, locals: { item_carousel_block: block }
                      ^
```
